corehq/apps/user_importer/importer.py

```diff
@@ -398,6 +398,15 @@ def format_location_codes(location_codes):
 
 def create_or_update_commcare_users_and_groups(upload_domain, user_specs, upload_user, group_memoizer=None,
                                                update_progress=None):
+    """"
+    Creates and Updates CommCare Users
+    For the associated web user username passed, for each CommCareUser
```

Comment:
Nitpick: the associated web user logic is a weird USH-only workflow, it's a little odd for it to be part of the primary function description.

Reply:
Not sure if it should not be present in the documentation because it's specific to a project unless we can extract all that logic to a method and put this part there, though i would still prefer this being here so that its clear that one should think about this workflow if they are making changes to this method.

Surprised, that this is not behind a feature flag.

Reply:
Fair.

It's a "secret" feature, since to access it you have to know about a magical user data key.